style.css

```diff
@@ -0,0 +1,826 @@
+    font-size: 20px;
+    margin: 0 0 0 205px;
+    padding-bottom: 100px;
+  }
+
+  #about {
+    background-image: url(images/program-bg.jpg);
+    margin: 0 0 0 -7.5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    right: 0;
+  }
+
+  #about-title {
+    color: #f9f9f9;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    width: 100%;
+    text-align: center;
+    padding-top: 25px;
+  }
+
+  .separator {
+    background-color: rgba(251, 82, 45, 0.7);
+    height: 2px;
+    width: 38px;
+    margin: -8px 0 30px 0;
+  }
+
+  #desktop-itinerary {
+    display: flex;
+    flex-direction: row;
+    gap: 3px;
+    width: 80%;
+    margin-bottom: 100px;
+  }
+
+  .itinerary {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-content: center;
+    background-color: rgba(255, 255, 255, 0.1);
+    width: 20%;
+    padding: 8px;
+    margin: 10px 0;
+  }
+
+  .itinerary img {
+    width: 50px;
+    height: 50px;
+    margin: 30px 0 -10px 36%;
+  }
+
+  #join-btn {
+    display: none;
+  }
+
+  #features {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  #features h3 {
+    color: #43403f;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+  }
+
+  #gamers {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-content: center;
+    width: 100%;
+    gap: 20px;
+    margin: 30px 0 50px 0;
+  }
+
+  .gamer-entry {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    width: 30%;
+    padding-left: 100px;
+    margin-bottom: 30px;
+  }
+
+  #chess-bg {
+    width: 60px;
+    height: 60px;
+    float: left;
+    margin: -15px -65px 0 0;
+  }
+
+  .gamer-entry img {
+    width: 120px;
+    height: 120px;
+  }
+
+  .country {
+    color: #fb522d;
+    margin-top: 4px;
+    font-weight: bold;
+  }
+
+  #more-btn {
+    display: none;
+  }
+
+  .partners.about {
+    display: none;
+  }
+
+  .partners {
+    background-color: #272a31;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  #partner-list {
+    display: flex;
+    gap: 80px;
+    justify-content: space-between;
+    margin: 30px 0 50px 0;
+  }
+
+  .game-logo {
+    width: 50px;
+    height: 50px;
+  }
+
+  #logo-txt {
+    color: #43403f;
+    text-align: center;
+    margin: 0 30% 40px 30%;
+  }
+
+  #logo-2 {
+    border-radius: 50%;
+  }
+
+  #pastevents {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 20px;
+  }
+
+  .pastlog {
+    width: 300px;
+    height: 200px;
+    margin-bottom: 50px;
+  }
+
+  #first {
+    background-image: url(images/Past-Fest-2021.jpg);
+  }
+
+  .overlay {
+    width: 300px;
+    height: 200px;
+    padding: 0;
+    background: rgba(251, 82, 45, 0.7);
+  }
+
+  #second {
+    background-image: url(images/Past-Fest-2022.jpg);
+    background-size: contain;
+  }
+
+  footer {
```

Comment:
🌸  Your code is really good thus far, consider using a flexbox to display the footer elements according to the design